### DISCUSSION TITLE
Using intellichlor as flow meter

### DISCUSSION THREAD

**Discussion by jmorris644** [tagyoureit/nodejs-poolController#492] [General]
Every so often when my pump starts I am getting an air gap so the water does not flow. I am hoping to use a combination of the pump being "on" and the status of the intellichlor to let me know if I have a flow issue. I am using MQTT into node-red for all of my automations. I have looked at all of the MQTT outputs from the intellechlor and do not see one that actually tells me if it is running or has a flow issue. I see that the dashPanel is able to identify when the intellichlor is not running. Is there some MQTT status that I am missing?

**Comment by rstrouse** [comment]
Unfortunately the outputs vary between chlorinator models even from the same manufacturer.  The reliability of low flow detection reporting from the cell isn’t as good as one might hope but if the cell is reporting it you will find it to be on the status leaf for the chlorinator.

**Reply by jmorris644** [reply]
My apologies. I do not know what a "leaf" is.

For development purposes I am capturing all MQTT messages for the chlorinator via this topic 

`easytouch2-4p/state/chlorinators/1/intellichlor--20/#`

The `status` returned from this list of topics is

`{"id":1,"isOn":"off"}`

The chlorinator is actually running.

I do see an currentOutput of 25, which is what I have it set at. Maybe that will go to zero if it is shut off?

**Reply by rstrouse** [reply]
Yes it will go to 0 on current output when it is not running.  That status should change to reflect whichever status values are supported by the chlorinator.
